gridcapa-swe-commons/src/test/java/com/farao_community/farao/gridcapa_swe_commons/shift/SweNetworkShifterTest.java

```diff
@@ -48,14 +49,16 @@
 /**
  * @author Mohamed Ben-rejeb {@literal <mohamed.ben-rejeb at rte-france.com>}
  */
-@SpringBootTest(classes = ProcessConfiguration.class)
+@SpringBootTest(classes = {ProcessConfiguration.class, CountryBalanceComputation.class})
```

Comment:
We still need this even with the @ComponentScan on the Main ?

Reply:
yes we do because the component scan is in another module